src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -171,13 +171,11 @@ class CrossDeviceLinkUI extends Component {
     const invalidNumber = !this.state.validNumber
     return (
       <div>
-        <div className={style.header}>
-          { error.type ?
-            <SmsError error={error} trackScreen={this.props.trackScreen}/> :
-            <h1 className={`${theme.title} ${style.title}`}>Continue verification on your mobile</h1> }
-        </div>
+        { error.type ?
+          <SmsError error={error} trackScreen={this.props.trackScreen}/> :
+          <h1 className={`${theme.title} ${style.title}`}>Continue verification on your mobile</h1> }
         <div className={theme.thickWrapper}>
-          <div>We’ll text a secure link to your mobile</div>
+          <p>We’ll text a secure link to your mobile</p>
```

Comment:
necessary? using <p> should be tested on multiple browsers. also is more likely customers has style that will interfere with `p` than with `div`

Reply:
yeah I will change it because https://github.com/onfido/onfido-sdk-ui/pull/269/files introduces some styling anyway